wave_vision/tests/matcher_tests/brute_force_tests.cpp

```diff
@@ -0,0 +1,137 @@
+// C++ Headers
+#include <algorithm>
+
+// Libwave Headers
+#include "wave/wave_test.hpp"
+#include "wave/vision/detector/fast_detector.hpp"
+#include "wave/vision/descriptor/brisk_descriptor.hpp"
+#include "wave/vision/matcher/brute_force_matcher.hpp"
+
+namespace wave {
+
+const auto TEST_CONFIG = "tests/config/matcher/brute_force_matcher.yaml";
+const auto TEST_IMAGE_1 = "tests/data/image_center.png";
+const auto TEST_IMAGE_2 = "tests/data/image_right.png";
+
+// Test fixture to load image data
+class BFTest : public testing::Test {
+ protected:
+    BFTest() {
+        initDetector();
```

Comment:
Since your class definition lists `FASTDetector fast;`, it was already _initialized_ (`FASTDetector()` was called) before you _assign_ to it here.